tests/conftest.py

```diff
@@ -151,6 +161,19 @@ def storageclass_factory_class(
     )
 
 
+@pytest.fixture(scope='session')
+def storageclass_factory_session(
+    request,
+    ceph_pool_factory_session,
+    secret_factory_session
```

Comment:
Same. Please try and used the non-admin secrets created in openshift-storage for adding in SC. 

$ oc get secret -n openshift-storage|grep csi|grep 'kubernetes.io/rook'
rook-csi-cephfs-node                                        kubernetes.io/rook                    2      25h
rook-csi-cephfs-provisioner                                 kubernetes.io/rook                    2      25h
rook-csi-rbd-node                                           kubernetes.io/rook                    2      25h
rook-csi-rbd-provisioner                                    kubernetes.io/rook                    2      25h

Reply:
Ok, but this should be changed in a different pull request as this is a thing that can affect more test cases and shouldn't be part of a upgrade test case PR.